rpcs3/Emu/Cell/PPUThread.cpp

```diff
@@ -2099,17 +2126,72 @@ extern void ppu_precompile(std::vector<std::string>& dir_queue)
 				continue;
 			}
 
+			std::string upper = fmt::to_upper(entry.name);
+
 			// Check .sprx filename
-			if (fmt::to_upper(entry.name).ends_with(".SPRX"))
+			if (upper.ends_with(".SPRX"))
 			{
+				// Skip already loaded modules or HLEd ones
```

Comment:
I only partially understand this part. If they are already loaded, ok. But if they are HLEd, we used to still compile them.

Reply:
I tried to compile them as well, but it messed up their HLE into non-existing LLE (which crashed of course), I think it's a bug/unimplemented feature of ppu_unload_prx.